pkg/nsx/services/vpc/vpc.go

```diff
@@ -625,6 +625,10 @@ func (service *VPCService) CreateOrUpdateAVIRule(vpc *model.Vpc, namespace strin
 	if !enableAviAllowRule {
 		return nil
 	}
+	if !nsxutil.IsLicensed(nsxutil.FeatureDFW) {
+		log.Info("avi rule cannot be created or updated due to the absence of a DFW license")
```

Comment:
`absence of a DFW license`: Use `no DFW license` as line 141?, or line 141 keeps the same with this log message